mocks/wiremock/templates/managedOrganisationByEmail.json

```diff
@@ -0,0 +1,24 @@
+{
+  "priority": 1,
+  "request": {
+    "customMatcher" : {
+      "name" : "jwt-matcher",
+      "parameters" : {
+        "payload": {
+          "sub" : "[email]"
+        },
+        "request" : {
+          "url" : "/refdata/internal/v1/organisations",
+          "method" : "GET"
+        }
```

Comment:
I think is was supposed to work a bit differently.
the endpoint GET /refdata/internal/v1/organisations accepts as query param id of organisation (e.g ORGSA, ORGSN) and returns organisation details.
Endpoint will be accessible by system user, what it means that response does not depends on jwt token

it should be something like:
`
{
  "mappings": [
    {
      "request": {
        "method": "GET",
        "urlPath": "/refdata/internal/v1/organisations",
        "queryParameters": {
          "id": {
            "equalTo": "ORGHN"
          }
        }
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json"
        },
        "bodyFileName": "prd/hillingdonOrganisation.json"
      }
    },
    {
      "request": {
        "method": "GET",
        "urlPath": "/refdata/internal/v1/organisations",
        "queryParameters": {
          "id": {
            "equalTo": "ORGEXT"
          }
        }
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json"
        },
        "bodyFileName": "prd/solicitorsOrganisation.json"
      }
    },
    {
      "request": {
        "method": "GET",
        "urlPath": "/refdata/internal/v1/organisations",
        "queryParameters": {
          "id": {
            "equalTo": "ORGSA"
          }
        }
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json"
        },
        "bodyFileName": "prd/swanseaOrganisation.json"
      }
    },
    {
      "request": {
        "method": "GET",
        "urlPath": "/refdata/internal/v1/organisations",
        "queryParameters": {
          "id": {
            "equalTo": "ORGSW"
          }
        }
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json"
        },
        "bodyFileName": "prd/swindonOrganisation.json"
      }
    },
    {
      "request": {
        "method": "GET",
        "urlPath": "/refdata/internal/v1/organisations",
        "queryParameters": {
          "id": {
            "equalTo": "ORGSNW"
          }
        }
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json"
        },
        "bodyFileName": "prd/wiltshireOrganisation.json"
      }
    }
  ]
}

`